docs/rules/forbid-component-props.md

```diff
@@ -47,7 +47,7 @@ Each array element can either be a string with the property name or object speci
 ```js
 {
   "propName": "someProp",
-  "allowedFor": [SomeComponent, AnotherComponent]
+  "allowedFor": [SomeComponent, "AnotherComponent", "/^Svg/"]
```

Comment:
i have a strong aversion to allowing users to provide regexes; would allowing `*` be sufficient?

Reply:
I think that would be flexible enough. May I ask why you don't like it more open-ended?

Reply:
Also, are you referring to allowing wildcard only at the beginning of the string? End? Both? Multiple? E.g. `*Icon*`

Reply:
I think at any position would be fine - altho i'm not sure how complex this makes the algorithm.

It'd also be fine to start with just "at the beginning" and/or "at the end", to be simpler, and allow use only of startsWith/endsWith.

Reply:
Escaping regular expressions is a very very hard problem, and passing user input into the RegExp constructor will likely be a bug farm, where the answer used to close the issue will be "that's not a valid regex", over and over.